unique_toolkit/unique_toolkit/content/service.py

```diff
@@ -112,9 +110,9 @@ def _trigger_search_content_chunks(
                 chatId=self.state.chat_id,
                 searchString=search_string,
                 searchType=search_type.name,
-                scopeIds=scope_ids,
+                scopeIds=scope_ids,  # type: ignore
                 limit=limit,
-                reranker=reranker_config,
+                reranker=reranker_config.model_dump() if reranker_config else None,  # type: ignore
```

Comment:
we should not introduce any further type ignores. we should rather update then also the sdk